allennlp/nn/util.py

```diff
@@ -1289,3 +1289,30 @@ def add_positional_features(tensor: torch.Tensor,
 def clone(module: torch.nn.Module, num_copies: int) -> torch.nn.ModuleList:
     "Produce N identical layers."
     return torch.nn.ModuleList([copy.deepcopy(module) for _ in range(num_copies)])
+
+
+def make2d(inputs: torch.Tensor) -> torch.Tensor:
```

Comment:
These should handle the 1D case better - if you give it a 1D tensor, it does not make it 2D, despite it's name - maybe it should error or unsqueeze or something?

Reply:
it needs to handle the 1d case, so I renamed it